docs/deployment.md

```diff
@@ -23,7 +32,33 @@ resource_registry:
   OS::TripleO::Services::NovaCompute: OS::Heat::None
 
 parameter_defaults:
-  NeutronMechanismDrivers: [openvswitch, baremetal]
+  NeutronMechanismDrivers: [ovn, baremetal]
```

Comment:
I don't think we can recommend ovn, now that we know that neutron dhcp won't work with it.